lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1153,16 +1153,18 @@ def _create_node_arm_parameters(
         arm_parameters = AzureNodeArmParameter.from_node_runbook(runbook)
 
         os_disk_size = 30
-        if arm_parameters.vhd:
+        if arm_parameters.osdisk_size_in_gb is not None:
+            pass
+        elif arm_parameters.vhd:
```

Comment:
If both `osdisk_size_in_gb` and `vhd` are defined in runbook, this logic might fail to run `_get_deployable_vhd_path` which I believe is required for vhd deployments.